### ISSUE TITLE
Make a multiplayer snake game using Sockets/Networking and PyGame. 

### ISSUE THREAD

**Issue by nikhil25803** [TusharAMD/Runn#13]
Hey @TusharAMD 
I am a GSSoC participant, can I work on this?

**Comment by nikhil25803** [comment]
I really want to contribute to this project, so please assign it to me.

**Comment by TusharAMD** [comment]
Yes sure awesome idea. You can start working. Will that include gesture control? If no then too there is no issue, your project will be enhanced later. Assigning this issue to you.
